Gordon360/Models/CCT/Context/CCTContext.cs

```diff
@@ -754,13 +797,28 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                 entity.Property(e => e.SNID).ValueGeneratedNever();
             });
 
+            modelBuilder.Entity<SuperAdmin>(entity =>
+            {
+                entity.HasOne(d => d.usernameNavigation)
+                    .WithOne(p => p.SuperAdmin)
+                    .HasForeignKey<SuperAdmin>(d => d.username)
+                    .OnDelete(DeleteBehavior.ClientSetNull)
+                    .HasConstraintName("FK_SuperAdmin_Participant");
+            });
+
```

Comment:
What is this table for? Super Admin is an AD group.

Reply:
Ahh, this was a remnant before I learned how to do AD groups. I'll remove it.